lib/geo/postgis.ex

```diff
@@ -28,7 +28,11 @@ if Code.ensure_loaded?(Ecto.Query) do
     end
 
     defmacro st_distance_sphere(geometryA, geometryB) do
-      quote do: fragment("ST_Distance_Sphere(?,?)", unquote(geometryA), unquote(geometryB))
+      quote do: fragment("ST_DistanceSphere(?,?)", unquote(geometryA), unquote(geometryB))
```

Comment:
Looks like it changed in PostGIS v2.2.0: http://postgis.net/docs/manual-2.2/ST_DistanceSphere.html

> Changed: 2.2.0 In prior versions this used to be called ST_Distance_Sphere